src/status_im/contexts/wallet/send/routes/view.cljs

```diff
@@ -87,160 +80,164 @@
-                                                            :market-values-per-currency
-                                                            currency
-                                                            :price)
-             {token-balance :total-balance}             (rf/sub [:wallet/token-by-symbol
-                                                                 (str token-symbol)
-                                                                 [chain-id]])
-             current-crypto-limit                       (utils/get-standard-crypto-format
-                                                         token
-                                                         token-balance)
-             current-fiat-limit                         (.toFixed (* token-balance conversion-rate) 2)
-             current-limit                              (if crypto-currency?
-                                                          current-crypto-limit
-                                                          current-fiat-limit)
-             crypto-decimals                            token-decimals
-             input-amount                               (controlled-input/input-value input-state)
-             [is-amount-locked? set-is-amount-locked]   (rn/use-state (some? locked-amount))
-             bottom                                     (safe-area/get-bottom)
-             amount-in-crypto                           (if crypto-currency?
-                                                          input-amount
-                                                          (number/remove-trailing-zeroes
-                                                           (.toFixed (/ input-amount
-                                                                        conversion-rate)
-                                                                     crypto-decimals)))
-             locked-greater-then-send-amount?           (let [amount      (money/bignumber
-                                                                           amount-in-crypto)
-                                                              send-amount (money/bignumber
-                                                                           send-amount-in-crypto)]
-                                                          (and (money/bignumber? amount)
-                                                               (money/bignumber? send-amount)
-                                                               (money/greater-than amount send-amount)))
-             swap-between-fiat-and-crypto               (fn [swap-to-crypto-currency?]
-                                                          (set-crypto-currency swap-to-crypto-currency?)
-                                                          (set-input-state
-                                                           (fn [input-state]
+#_(defn- edit-amount
```

Comment:
This code is not used in production anyway because we decided to get rid of editing network values before release. Also since we are going to simplify send flow, most probably it wont't be used. It reuses `token-input` but I decided to not bother fixing it. I believe we can fully remove it but better to do it after final decision of `send` simplification.